src/client/pps/pps.proto

```diff
@@ -178,6 +179,7 @@ message JobInfo {
   pfs.Commit stats_commit = 29;
   bool enable_stats = 32;
   string salt = 33;
+  bool batch = 34;
```

Comment:
I doesn't actually implement the logic (as you've no doubt seen) I added this flag before I split off the other branch to implement user space batching and figured it wasn't really worth rewriting git history to get rid of it.